daemon/logger/copier_test.go

```diff
@@ -102,6 +106,50 @@ func TestCopier(t *testing.T) {
 	}
 }
 
+// TestCopierTimestamp tests that Copier is using a time function to add timestamp to a message
+func TestCopierTimestamp(t *testing.T) {
+	timeFunc := func() time.Time {
+		return time.Date(2017, time.June, 0, 0, 0, 0, 0, time.UTC)
+	}
+	var stdout bytes.Buffer
+	if _, err := stdout.WriteString("hello world\n"); err != nil {
+		t.Fatal(err)
+	}
+	var jsonBuf bytes.Buffer
+
+	jsonLog := &TestLoggerJSON{Encoder: json.NewEncoder(&jsonBuf)}
+
+	c := NewCopier(
+		map[string]io.Reader{
+			"stdout": &stdout,
+		},
+		jsonLog, timeFunc)
```

Comment:
The formatting here is kind of strange. Maybe just make it a single line? Or one argument per line (the map can be a single line)